spec/amp-cors-requests.md

```diff
@@ -50,10 +52,14 @@ The resulting HTTP response has to also contain the following headers:
  - `AMP-Access-Control-Allow-Source-Origin: <source-origin>`. Here "source-origin" indicates the source origin that is allowed to read the authorization response as was verified via "__amp_source_origin" URL parameter. Ex: "https://publisher1.com".
  - `Access-Control-Expose-Headers: AMP-Access-Control-Allow-Source-Origin`. This header simply allows CORS response to contain the "AMP-Access-Control-Allow-Source-Origin" header.
 
-#### Note on non-idempotent Requests
+Note that AMP CORS never requires preflighted requests so it's safe to assume that requests with the `OPTIONS` method are **not** coming from a legitimate AMP page and an error response should be returned.
```

Comment:
When making `POST` requests with `Content-Type: application/json`, currently that only happens in `amp-user-notifications` dismissal logic. Forms don't do that since we're using form data content type for `POST` requests.

We filed an issue to figure what to do there #4910